seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -403,9 +403,10 @@
       "id": "a3f503de51cfab748ff001aa",
       "title": "Pairwise",
       "description": [
-        "Return the sum of all element indices of array <code>arr</code> that can be paired with one other element to form a sum that equals the value in the second argument <code>arg</code>. If multiple sums are possible, return the smallest sum. Once an element has been used, it cannot be reused to pair with another.",
-        "For example, <code>pairwise([1, 4, 2, 3, 0, 5], 7)</code> should return <code>11</code> because 4, 2, 3 and 5 can be paired with each other to equal 7 and their indices (1, 2, 3, and 5) sum to 11.",
-        "<code>pairwise([1, 3, 2, 4], 4)</code> would only return <code>1</code>, because only the first two elements can be paired to equal 4, and the first element has an index of 0!",
+        "Given an array <code>arr</code>, find element pairs whose sum equal the second argument <code>arg</code> and return the sum of their indices.",
+        "If multiple element pairs with different indices are possible (e.g. there are two pairs of <code>(0, 1)</code> that sum to <code>1</code> but one pair with indices <code>0</code> and <code>4</code>, and another with <code>1</code> and <code>4</code>), return the smallest sum (e.g. <code>4 = 0 + 4</code>). Once an element has been used, it cannot be reused to pair with another.",
```

Comment:
Yikes, this is scary.

What might make it a bit easier is if you use example numbers that are higher than the indexes.

IE:
`pairwise([7, 9, 11, 13, 15], 20)` - Then you can very clearly see that the "indexes" sum to 6 (I think), but that they are in no way related to the numbers listed.
When you do something with low numbers it can be confusing because it is not clear if you're not talking about the index or the value.  Using these larger numbers may clarify that.

The other possibility would be to add a table which had a header of the indexes and a row of the values:

Index | 0 | 1 | 2 | 3 | 4
--- | --- | --- | --- | --- | ---
Value  | 7|9|11|13|15

Then you show:
7 + 13 = 20, indexes **0** + **3** = 3
9 + 13 = 20, indexes **1** + **2** = 3
3 + 3 = 6